From: =?utf-8?q?Lisandro_Dami=C3=A1n_Nicanor_P=C3=A9rez_Meyer?=
 <lisandro@debian.org>
Date: Wed, 13 Apr 2016 13:53:28 +0200
Subject: use_always_present_path_to_test
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

Description: use a path that's always there
 Or at least in Debian ;)
 .
 To try to use /etc/cups/ it will require to install cups first. But to be
 able to achieve the same result one can simply use /etc/crond.d, which
 is present both in Debian and Red Hat almost always (if not always).
Author: Lisandro Damián Nicanor Pérez Meyer <lisandro@debian.org>
Forwarded: https://bugs.kde.org/show_bug.cgi?id=330107
Last-Update: 2014-01-17
---
 kioslave/trash/tests/testtrash.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/kioslave/trash/tests/testtrash.cpp b/kioslave/trash/tests/testtrash.cpp
index f99f0f0..bb4e87f 100644
--- a/kioslave/trash/tests/testtrash.cpp
+++ b/kioslave/trash/tests/testtrash.cpp
@@ -894,8 +894,8 @@ void TestTrash::moveDirectoryFromTrash()
 void TestTrash::trashDirectoryOwnedByRoot()
 {
     KUrl u;
-    if ( QFile::exists( "/etc/cups" ) )
-        u.setPath( "/etc/cups" );
+    if ( QFile::exists( "/etc/cron.d" ) )
+        u.setPath( "/etc/cron.d" );
     else if ( QFile::exists( "/boot" ) )
         u.setPath( "/boot" );
     else
